projects/demo/src/modules/components/sheet-dialog/examples/1/index.ts

```diff
@@ -21,7 +21,7 @@ export class TuiSheetDialogExample1 {
                 switchMap(() =>
                     inject(TuiSheetDialogService).open('', {label: 'Simple sheet'}),
                 ),
-                takeUntil(inject(TuiDestroyService, {self: true})),
+                takeUntilDestroyed(inject(DestroyRef)),
```

Comment:
remove empty providers, replace takeUntilDestroyed(inject(DestroyRef)) and remove DestroyRef from imports